### ISSUE TITLE
questions about two-class supervised training

### ISSUE THREAD

**Issue by cxcx1996** [amazon-research/spot-diff#8]
1. What kinds of data augment are used in two-class supervised training?

2. Will you release the model.py of U-Net, which is used in two-class segmentation?  (it is helpful to fair comparison)

**Comment by yzou2** [comment]
A1: for two-class supervised learning, we only use horizontal random flipping (prob=0.5) as we do not want to introduce bias to certain defect types. 
A2: For Unet, we leverage unet in this package: https://github.com/qubvel/segmentation_models.pytorch

**Comment by cxcx1996** [comment]
> A1: for two-class supervised learning, we only use horizontal random flipping (prob=0.5) as we do not want to introduce bias to certain defect types. A2: For Unet, we leverage unet in this package: https://github.com/qubvel/segmentation_models.pytorch

Thanks!

**Comment by cxcx1996** [comment]
> A1: for two-class supervised learning, we only use horizontal random flipping (prob=0.5) as we do not want to introduce bias to certain defect types. A2: For Unet, we leverage unet in this package: https://github.com/qubvel/segmentation_models.pytorch

Thanks! And 
1. What's the training batchsize and h,w (for cls and seg task respectively)? 
2. The AU-PR in few-shot 2-class classification is quite lower than AU-PR in 1-class classification in paper. What's the reason for that?

**Comment by yzou2** [comment]
Sorry for the late response. The training batch size is 8. You can try to use 800x800 if possible. The resolution is important especially for segmentation. For two-class setup, you can try to use focal loss to address the imbalance problem.

**Comment by cxcx1996** [comment]
> Sorry for the late response. The training batch size is 8. You can try to use 800x800 if possible. The resolution is important especially for segmentation. For two-class setup, you can try to use focal loss to address the imbalance problem.

Thanks!
